### PR TITLE
GL-384: Add deleting pseudo measures

### PR DESCRIPTION
**Pull request by rasikasri**
### Jira link

[GL-384](https://transformuk.atlassian.net/browse/GL-384)

### What?

I have added/removed/altered:

- [ ] Added GL api to delete measures

### Why?

I am doing this because:

- Admin user should be able to manage GL measures

### CODE REVIEW

**Review by shjohnson — APPROVED** [review]
LGTM

**Review by shjohnson — APPROVED** [review]
LGTM

File: app/controllers/api/admin/green_lanes/category_assessments_controller.rb [hunk 1]
```diff
@@ -8,7 +8,10 @@ class CategoryAssessmentsController < AdminController
         before_action :check_service, :authenticate_user!
 
         def index
-          render json: serialize(category_assessments.to_a, pagination_meta)
+          options = { is_collection: true }
```
Comment (alexdesi) [inline app/controllers/api/admin/green_lanes/category_assessments_controller.rb:11]: a cleaner version:
```
 options = { 
   is_collection: true ,
   include: %i[theme],
   meta: pagination_meta(category_assessments)
}
```

File: app/controllers/api/admin/green_lanes/measures_controller.rb [hunk 2]
```diff
@@ -20,20 +20,29 @@ def index
         end
 
         def show
+          options = { is_collection: false }
```
Comment (alexdesi) [inline app/controllers/api/admin/green_lanes/measures_controller.rb:23]: options = {
          is_collection: false,
          include: %i[category_assessment]
          }

**Pull request merged**